README.md

```diff
@@ -1,24 +1,116 @@
-application up and running.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
 
-Things you may want to cover:
+# 📖 Firmtecs_blog <a name="about-project"></a>
 
-* Ruby version
+> This Firmtecs_blog app will be a classic example of a blog website. I created a fully functional website that will show the list of posts and encourage readers to interact with them by adding comments and liking posts.
 
-* System dependencies
+Please click on the video below 👇 for more explanation
 
-* Configuration
+## Explanation video
 
-* Database creation
+[]()
 
-* Database initialization
+## 🛠 Built With <a name="built-with"></a>
 
-* How to run the test suite
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+    <li><a href="https://www.postgresql.org/">Postgres</a></li>
+  </ul>
 
-* Deployment instructions
+## 💻 Getting Started <a name="getting-started"></a>
 
-* ...
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- IDE or code editor: **VsCode, atom, sublime**...
+- To have [Ruby](https://www.ruby-lang.org/en/), [RoR gem](https://rubyonrails.org/) and [Git](https://git-scm.com/) installed on your system.
+
+### Setup
+
+Clone this project on your local machine using the following command on your terminal:
+
+```sh
+  git clone https://github.com/blas147/Firmtecs_blog.git
+```
+
+### Install
+
+Run the following command to install all project's dependencies:
+
+```sh
+bundle install
+```
+
+Run the following command to install Rspec for tests:
+
+```sh
+gem install rspec
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails s
+```
+
+## Run Tests
+
+Run the following command to catch all style errors:
+
+```sh
+rubocop
+```
+
+Run the following command for the rspec tests(with details):
+
+```sh
+rspec ./spec --format documentation
+
+```
+
+## 👥 Author <a name="authors"></a>
+
+👤 **Chukwuma Paul Mosanya**
+
+[GitHub](https://github.com/blase147) | [Twitter](https://twitter.com/ChukwumaMosanya) | [LinkedIn](https://www.linkedin.com/in/chukwuma-mosanya)
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/blase147/creating_data_model/issues/new).
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give this project a star if you like it !
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly make sure to add a LICENSE to your repo so this link is working properly.